packages/indexer-agent/src/network.ts

```diff
@@ -557,15 +564,39 @@ export class Network {
         {
           indexer: this.indexerAddress.toLocaleLowerCase(),
           disputableEpoch,
-          minimumQueryFeesCollected: this.queryFeesCollectedClaimThreshold.toString(),
+          minimumQueryFeesCollected: this.rebateClaimMinSingleValue.toString(),
         },
       )
 
       if (result.error) {
         throw result.error
       }
 
-      return result.data.allocations.map(parseGraphQLAllocation)
+      const totalFees: BigNumber = result.data.allocations.reduce((total: BigNumber, rawAlloc: { queryFeesCollected: string }) => {
+        return total.add(BigNumber.from(rawAlloc.queryFeesCollected))
+      }, BigNumber.from(0))
+
+      const parsedAllocs: Allocation[] = result.data.allocations.map(parseGraphQLAllocation)
```

Comment:
Move this below the next block so it only executes if the batch value is above the threshold?